File: Pratica 3 - Estrutura de Repeticao/cpp_logic/desafio5.cpp
#include <iostream>
using namespace std;

int main() {

    cout << "Digite o nome do usário: ";
    string nome;
    cin >> nome;
    string senha_correta = "senha123";
    string senha = "";

    while (senha != senha_correta) {
        cout << "Digite a senha: ";
        cin >> senha;
    }

    cout << "Bem-vindo, " << nome << "!" << endl;

    return 0;
}

File: Pratica 3 - Estrutura de Repeticao/java_logic/Desafio5.java
import java.util.Scanner;

public class Desafio5 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Digite o nome de usuário: ");
        String usuario = sc.nextLine();
        String senha_correta = "senha123";
        String senha = "";

        while (!senha.equals(senha_correta)) {
            System.out.print("Digite a senha: ");
            senha = sc.nextLine();
        }

        System.out.println("Bem-vindo, " + usuario + "!");
        sc.close();
    }
}
